consensus/checks.go

```diff
@@ -239,11 +239,11 @@ func (consensus *Consensus) onViewChangeSanityCheck(recvMsg *FBFTMessage) bool {
 			Msg("[onViewChange] ViewChanging ID Is Low")
 		return false
 	}
-	if recvMsg.ViewID-consensus.current.ViewID() > MaxViewIDDiff {
+	if consensus.current.ViewID() != 0 && recvMsg.ViewID-consensus.current.ViewID() > MaxViewIDDiff {
```

Comment:
better fix in the right way. set current.viewID along with consensus.viewID's setting. In here:

// SetViewID set the viewID to the height of the blockchain
func (consensus *Consensus) SetViewID(height uint64) {
	consensus.viewID = height
}